templates/explore/code.tmpl

```diff
@@ -5,7 +5,7 @@
 		<form class="ui form ignore-dirty" style="max-width: 100%">
             <input type="hidden" name="tab" value="{{$.TabName}}">
             <div class="ui fluid action input">
-                <input name="q" value="{{.Keyword}}" placeholder="{{.i18n.Tr "explore.search"}}..." autofocus>
+                <input name="q" value="{{.Keyword}}" placeholder="{{.i18n.Tr "explore.search"}}...">
```

Comment:
Didn't this is a real search page? Maybe we should keep it on this file.

Reply:
I guess we can keep it on the explore pages, I just want to eliminate it on the issue/pr pages.